### PR TITLE
remove console warnings, add angular http call

### LINKED ISSUE #19: type warning in console due to use of any
screen shot 2017-08-11 at 8 22 57 am

relates to #5

### PR DESCRIPTION
**Pull request by cxoltero**
resolves #19 

It also adds an [angular-http](https://angular.io/guide/http) call with observables to our gravatar service

### CODE REVIEW

File: src/services/gravatar.service.ts [hunk 1]
```diff
@@ -17,20 +17,22 @@ interface GravatarInterface {
...
 
+    // TODO, in order of http to work, please enable CORS
     return this.http.get(url)
-      .map((res: Response) => {
-        console.log(res, 'response');
-        return  res.json();
-      })
-      .catch((error: any) => Observable.throw(error.json().error || 'Server error'));
-
+               .map((res: Response) => {
```
Comment (thescientist13) [inline src/services/gravatar.service.ts]: indentation is off here, please rebase and fix after #24 is merged

Comment (thescientist13) [inline src/services/gravatar.service.ts]: all set ^

File: src/services/gravatar.service.ts [hunk 2]
```diff
@@ -17,20 +17,23 @@ interface GravatarInterface {
...
+  constructor(private http: Http) {
+  }
 
-  public getAvatar(email: string): Observable <any> {
+  public getAvatar(email: string) {
     const emailHash: string = Md5.hashStr(email).toString();
-    const url: string = `${this.gravatarUrl}${emailHash}.json`;
+    const url: string = `${GravatarService.gravatarUrl}${emailHash}.json`;
 
+    // TODO, in order of http to work, please enable CORS
```
Comment (thescientist13) [inline src/services/gravatar.service.ts]: > // TODO, in order of http to work, please enable CORS

how?  is this actually needed by the user of the application?  No `jsonp` with Angular available?

Comment (thescientist13) [inline src/services/gravatar.service.ts]: Testing this locally, I don't even see an HTTP call being made.  Is that expected?

**Comment by cxoltero** [comment]
@thescientist13, All changes needed to switch from http call to jsonp are submitted @ #43df179

**Pull request merged**